<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RRS route editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
      <item>
       <widget class="QTreeWidget" name="treeProject">
        <column>
         <property name="text">
          <string>Route project</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="twMain">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabProperties">
         <attribute name="title">
          <string>Properties</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Route title</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leRouteTitle"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Route description</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="ptRouteDescription">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>450</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pbPropApply">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbPropCancel">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabProfile">
         <attribute name="title">
          <string>Profile editor</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Initial ordinate</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QSpinBox" name="sbInitOrdinate">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="maximum">
                   <number>40000000</number>
                  </property>
                  <property name="singleStep">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>m</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableWidget" name="twProfileData">
                <property name="minimumSize">
                 <size>
                  <width>650</width>
                  <height>0</height>
                 </size>
                </property>
                <column>
                 <property name="text">
                  <string>Linear coord, m</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Ordinate, m</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Lenght, m</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Inclination</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Radius, m</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="pbProfPrevElementAdd">
                <property name="text">
                 <string>Add previos</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbProfNextElementAdd">
                <property name="text">
                 <string>Add next</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbProfElementRemove">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pbProfOpen">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbProfSave">
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabRouteMap">
         <attribute name="title">
          <string>Route map editor</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_route"/>
    <addaction name="actionOpen_route"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionZDS_profile_import"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_route">
   <property name="text">
    <string>New route</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_route">
   <property name="text">
    <string>Open route</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save route</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close route</string>
   </property>
  </action>
  <action name="actionZDS_profile_import">
   <property name="text">
    <string>ZDS profile import</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
